add-interactive.c

```diff
@@ -58,14 +58,15 @@ struct item {
 };
```

Comment:
Comment about commit message: Mention that we accept full line as input now. 
This was originally your idea, and I've described this change in my patch series, so this is just reminder of that. :)

Reply:
I think I was wrong originally, as I had assumed that `git add -i` takes single keystrokes by default...

But it does not, so this is not the change in behavior that I thought it would be. Not sure that we should mention it, for that reason: the reader might think that we changed behavior and be puzzled when we did not.

Reply:
Okay, thanks for letting me know :)